basyx.aasdiscoveryservice/basyx.aasdiscoveryservice-backend/src/main/java/org/eclipse/digitaltwin/basyx/aasdiscoveryservice/backend/CrudAasDiscovery.java

```diff
@@ -56,7 +56,7 @@
  */
 public class CrudAasDiscovery implements AasDiscoveryService {
```

Comment:
The changes here are not necessarily related to the PR. See remark in common.pagination